workers/common/src/main/kotlin/common/env/NuGetGenerator.kt

```diff
@@ -30,7 +30,7 @@ import org.eclipse.apoapsis.ortserver.workers.common.env.definition.NuGetDefinit
 class NuGetGenerator : EnvironmentConfigGenerator<NuGetDefinition> {
     companion object {
         /** The name of the configuration file created by this generator. */
-        private const val TARGET = "NuGet.Config"
+        private const val TARGET = ".nuget/NuGet/NuGet.Config"
```

Comment:
Should this be made `internal` so it could be used in the tests instead of hard-coding the same string again?

Reply:
Proper file placement is checked in two kind of tests (unit and integration ones), so that's why this string is used twice.
Also - in case of sharing this value, in fact there should be separate test to check if string content is valid.